.azure/gpu-tests.yml

```
@@ -112,16 +120,17 @@ jobs:
 
     - bash: |
         set -e
+        echo $CUDA_VISIBLE_DEVICES
         python requirements/collect_env_details.py
-        python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu >= 2, f'GPU: {mgpu}'"
+        python -c "import torch ; mgpu = torch.cuda.device_count() ; assert mgpu == 2, f'GPU: {mgpu}'"
         python requirements/pytorch/check-avail-strategies.py
         python requirements/pytorch/check-avail-extras.py
       displayName: 'Env details'
 
     - bash: bash .actions/pull_legacy_checkpoints.sh
       displayName: 'Get legacy checkpoints'
 
-    - bash: python -m coverage run --source pytorch_lightning -m pytest
+    - bash: python -m coverage run --source pytorch_lightning -m pytest .
```

nit: this is the same

```suggestion
    - bash: python -m coverage run --source pytorch_lightning -m pytest
```